lib/extensions/print_pdf.py

```diff
@@ -409,11 +294,8 @@ def effect(self):
             realistic_color_block_svgs=realistic_color_block_svgs
         )
         print_server.start()
-
-        time.sleep(1)
-        open_url("http://%s:%s/" % (print_server.host, print_server.port))
-
-        app = wx.App()
-        info_frame = PrintInfoFrame(None, title=_("Ink/Stitch Print"), size=(450, 350), print_server=print_server)
-        info_frame.Show()
-        app.MainLoop()
+        time.sleep(0.5)
```

Comment:
Yeah, it is, unfortunately.  It's a bit of a hack to avoid having to do more hacks.  Here's the deal:

We don't know the port the server is listening on until the flask server successfully starts (`PrintPreviewServer.run`).  I'd love to use a `threading.Condition` so that the main thread can wait until the server is started up, but as far as I can tell, Flask doesn't provide for callbacks on startup out of the box without hacks.  You can get a callback during the first request, but that doesn't help here.

Waiting a half second is a little "magic", but it seems the easiest way to get the job done.  A comment seems like a good idea, so I'll add one.